structuralcodes/geometry/_geometry.py

```diff
@@ -109,23 +107,23 @@ def __init__(
         name: t.Optional[str] = None,
         group_label: t.Optional[str] = None,
     ):
-        """Initializes a PointGeometry object
-        The name and grouplabel serve for filtering
-        in a compound object. By default it
-        creates a new name each time.
+        """Initializes a PointGeometry object.
+
+        The name and group_label serve for filtering in a compound object. By
+        default it creates a new name each time.
 
         Arguments:
-            point: a couple of coordinates or e shapely Point object
-            diameter: the diameter of the point
-            material: the material for the point (this can be a Material or
-                    a ConstitutiveLaw)
-            density (Optional: default None) when a ConstitutiveLaw is passed
-                    as material, the density can be providen by this argument.
-                    When the material is a Material object the density is taken
-                    from the material.
-            name (Optional: default None): the name to be given to the object
-            group_label (Optional: default None): a label for grouping several
-        objects
+            point (Union(Point, ArrayLike)): A couple of coordinates or a
+                shapely Point object.
+            diameter (float): The diameter of the point.
+            material (Union(Material, ConstitutiveLaw)): The material for the
+                point (this can be a Material or a ConstitutiveLaw).
+            density (Optional(float)): When a ConstitutiveLaw is passed as
+                material, the density can be providen by this argument. When
+                the material is a Material object the density is taken from the
+                material.
+            name (Optional(str)): The name to be given to the object.
+            group_label (Optional(str)): A label for grouping several objects.
```

Comment:
Shouldn't we specify that default is None in docstrings?

Reply:
I agree, thanks for pointing out.